packages/aws-rfdk/lib/deadline/test/render-queue.test.ts

```diff
@@ -1134,7 +1227,14 @@ describe('RenderQueue', () => {
                   },
                 ],
               },
-              `" --render-queue "http://renderqueue.${ZONE_NAME}:8080" \n` +
+              '" --render-queue "http://',
+              {
+                'Fn::GetAtt': [
+                  'RenderQueueLB235D35F4',
+                  'DNSName',
+                ],
+              },
```

Comment:
this looks like it could this be converted to a more readable `isolatedStack.resolve(...)` expression. What do you think?

Reply:
Updated